ctapipe/core/provenance.py

```diff
@@ -86,6 +86,16 @@ def start_activity(self, activity_name=sys.executable):
         self._activities.append(activity)
         log.debug(f"started activity: {activity_name}")
 
+    def _get_current_or_start_activity(self):
+        if self.current_activity is None:
+            log.warning(
+                "No activity has been started yet."
+                " Provenance().start_activity(<name>) should be called explicitly."
+            )
+            self.start_activity()
```

Comment:
Since getting here is worth a warning, it is probably a good idea to give the activity a special name like "not-properly-started" to encourage developers to start them properly.

Reply:
After adding this warning, it popped up a lot in interactive use. I think this is over-the-top, I will remove the warning again and change to info.